### PR TITLE
Header updates

### PR DESCRIPTION
**Pull request by sfunk3**
**JIRA Issue**: [EXHIBIT-75](https://jirautk.atlassian.net/browse/EXHIBIT-75) & [EXHIBIT-76](https://jirautk.atlassian.net/browse/EXHIBIT-76)

What does this do?
==================

Following the Galston tech meeting on 8/10, I've made several adjustments to responsive styles for header (logo, libraries links) and navigation menu, including:

- Swapping the png logo for svg and making sure the height is brand compliant (not under 30px at the smallest)
- Adding links for account, search, and home. These are just links to those resources, including search. Mark and I discussed this as MVP for this project, so feedback would be appreciated.
- Setting breakpoints and corresponding responsive styles for mobile views (e.g. the header as a whole is a lot tighter going from tablet size down, the hover underline on menu items no longer stretches to 100% on mobile views, etc)

How should this be reviewed?
============================

Checkout the `header-updates` branch and test the site header at different breakpoints to make sure responsiveness is up to par. Review to decide if we're all agreed on what's included and how it's implemented.

Additional notes:
=================

Please request changes if needed. I plan to organize the sass structure a bit better but wanted to get eyes on the changes.

### CODE REVIEW

**Comment by markpbaggett** [comment]
@sfunk3 This seems fine to me, but I'll wait to see if @mathewjordan has feedback before merge.

**Review by markpbaggett — APPROVED** [review]
@sfunk3 this seems fine to me, but i'll wait on feedback from @mathewjordan before merge.

**Review by mathewjordan — CHANGES_REQUESTED** [review]
Edit - Delete this as a submitting prematurely.

File: src/sass/components/header.scss [hunk 1]
```diff
@@ -38,5 +38,273 @@
...
+    margin-right: 3rem;
+  }
+
+  .libraries-link {
+    line-height: 20px !important;
+  }
+
+  svg {
+    padding: 1rem 0;
+  }
```
Comment (mathewjordan) [inline src/sass/components/header.scss:180]: This type of thing will affect all `<svg>` elements in the virtual DOM. We don't want to do that. What svg is it? can we wrap it or give it a class?

File: src/components/yith/UniversalHeader.js [hunk 2]
```diff
@@ -7,8 +7,34 @@ class UniversalHeader extends Component {
...
+          </div>
+        </div>
+
+        <a className="libraries-link" href="https://lib.utk.edu">Libraries</a>
+        </div>
+
+        <div className="utk-lib-links">
+        <a href="https://utk.primo.exlibrisgroup.com/discovery/login?vid=01UTN_KNOXVILLE:01UTK&lang=en" target="_blank">My Account</a>
+        <div className="exhibits-search" >
+          <a href="https://utk.primo.exlibrisgroup.com/discovery/search?vid=01UTN_KNOXVILLE:01UTK&lang=en" target="_blank"><span>Search</span></a></div>
```
Comment (mathewjordan) [inline src/components/yith/UniversalHeader.js:35]: Why do we wrap this in `<div>` but not the other `<a>`'s?

File: src/sass/components/header.scss [hunk 3]
```diff
@@ -38,5 +38,273 @@
...
+
+  .utk-logo-exhibits {
+    svg {
+      padding-top: 0;
+      padding-bottom: 0;
+    }
+  }
+}
+
+  @media (max-width: 375px) {
```
Comment (mathewjordan) [inline src/sass/components/header.scss:282]: Why these and the breakpoint vars?

**Review by mathewjordan — CHANGES_REQUESTED** [review]
@sfunk3 A few things. 

- **Responsiveness and Sass structure** - I think this isn't quite there responsively. At smaller viewports the items do not collapse at all and instead wrap underneath eachother. I also have concerns with the redundancy of structure in each media query on `src/sass/components/header.scss`. In some cases we are using breakpoint vars `@media (max-width: $breakpoint-xs)` and in others we are using hardcoded widths `@media (max-width: 500px)`. The redundancy is also happening for many selectors for each media query. I feel like we can handle this in a much cleaner way, especially since we'll need to extend some of responsive strategy across the repo.   

image

image

- **UniversalHeader.js**, code base wise, this is not a Yith component and should not be under  `src/components/yith`. Yith are the components that interact with IIIF manifests and render them to the screen. I think this should move to `src/components`.

- There are some smaller things but I'll dig into those inline.

**Comment by sfunk3** [comment]
@mathewjordan I think this is in a good place for another quick review when you have a chance.

**Review by mathewjordan — COMMENTED** [review]
@sfunk3 The revamp of the responsive media queries look great.

I'm not suggesting any changes but I do have one question. Is the scope of this PR to make the header fully responsive? As it is now, I would not say that the header row is collapsing as expected. Do we address this here? or in another issue?

image

**Comment by sfunk3** [comment]
@mat Thanks for reviewing. My thinking is that we merge these changes in and address updates separately.

**Review by mathewjordan — APPROVED** [review]
Sounds good @sfunk3

**Pull request merged**